.github/workflows/build_test.yml

```diff
@@ -17,6 +17,7 @@ permissions:
 
 jobs:
   build:
+    if: github.repository == 'systemd/systemd' || github.repository == 'systemd/systemd-stable'
```

Comment:
I think it's OK to turn off flaky actions like mkosi but I'm not sure why for example the build test action should be disabled for example. Those things are used in places like https://github.com/redhat-plumbers/systemd-rhel10 for example.

Reply:
@mrc0mmand looking at https://github.com/redhat-plumbers/systemd-rhel10/pull/63 it seems the actions are patched there. If this PR is merged it should be necessary to patch them more to unblock them there (and in the other downstream forks).

Reply:
I'd like to keep this to make the CI disabled by default on forks.
People or projects that wants to run the CIs on their fork, it can be reenabled anytime they want.

Reply:
Right. Again, this is just a default setting, to make the actions not torture GitHub by forks.

Reply:
The actions have to be patched and those patches have to be rebased more often to keep them working. I don't think it should be necessary to just run the tests. With things like

![Screenshot 2025-04-16 at 13 34 24](https://github.com/user-attachments/assets/ab87257f-806c-4830-a7d9-23bf001b724b)

it's a non-issue.

Reply:
I forked mkosi and went to the actions tab:
![Screenshot 2025-04-16 at 13 49 11](https://github.com/user-attachments/assets/93e48f9d-7715-4755-927b-d1e81e048052)

so it looks like these days the actions aren't run by default.

Reply:
Either way now that Github appears to have switched to the meaningful default there I think it should be OK to guard actions that don't work with forks (like CIFuzz, coverage, coverity). Everything else should be OK.

(By the way it's already necessary to bring back stuff disabled upstream for no reason like https://github.com/systemd/systemd/issues/36880, maintain things where backtraces are actually symbolized and so on so it would be nice if wasn't even harder to just run the systemd tests).

Reply:
Thanks. So, this PR is mostly useless, except for forks older than the defaults-to-off rule applied. Closing.